tests/testdata/demo-modules/check-error/demo-module-4/can-load.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Can load function to determine if Site Health module is supported or not.
+ *
+ * @package performance-lab
+ */
+
+return static function () {
+    return new WP_Error( 'module_not_loaded', esc_html__( 'The module cannot be loaded with Performance Lab since it is disabled.', 'performance-lab' ) );
```

Comment:
Since this is just a demo for tests, we don't need to invent a reason why it cannot be loaded. IMO that makes the message less confusing :)
```suggestion
    return new WP_Error( 'cannot_load_module', esc_html__( 'The module cannot be loaded.', 'performance-lab' ) );
```

Reply:
Make sense. Thanks.